src/business/PostBusiness.ts

```diff
@@ -0,0 +1,49 @@
+import { PostsDatabase, POST_TYPE } from "../data/PostsDatabase"
+import { Authenticator } from "../services/Authenticator"
+import { IdGenerator } from "../services/IdGenerator"
+
+export class PostBusiness {
+
+  public async createPost (token: string, post_type: string, post_photo: string, post_description: string, post_createdAt: string) {
+    if (!post_type || !post_description || !post_photo){
+      throw new Error("Informe todos os dados!")
+    }
+
+    if(!post_type) {
+      post_type = POST_TYPE.NORMAL
+    }
```

Comment:
Muito legal deixar um valor default para o tipo de post! Uma outra forma de fazer isso é na hora de passar os parâmetros:

`public async createPost (token: string, **post_type: POST_TYPE = POST_TYPE.NORMAL**, post_photo: string, post_description: string, post_createdAt: string) {}`

Dessa forma, se não for passado nenhum valor de post_type, ele vai assumir o valor padrão que é o NORMAL.